oracle/onchain.go

```diff
@@ -64,6 +64,11 @@ type Onchain struct {
 	validators      map[phase0.ValidatorIndex]*v1.Validator
 }
 
+// ValidatorAccessor is an interface to access only the validators map
+type ValidatorAccessor interface {
```

Comment:
This interface seems a bit weird. Can we avoid it?

Reply:
IMO its the simplest way to access from oracle some elements of onchain (consensus client, etc) without importing the whole `onchain` object

Reply:
leftover, lets remove

Reply:
done